slides/src/content/introduction.md

```diff
@@ -37,4 +53,247 @@ FIXME
+- `styles` are css stylesheets that get applied to your component
+- `AppComponent` is a class that decides how the component should act
+
+---
+
+## What is a Component?
+
+- The core building block of an Angular application is the component
+- In Angular, a component is some chunk of application logic combined with something that gets displayed
+- Everything being displayed on any page should be associated with some component
+
+---
+
+## Component Template Syntax
+
+- `<h1>{{title}}</h1>` represents __interpolation__
+- Interpolation attempts to evaluate the expression between the curly braces and turn that into a string
+- Symbols (such as title) are evaluated within an expression context, in this case, the component's class members
+
+---
+
+## Generating a New Component
+
+- We'll use Angular CLI to create a new component
+- `ng generate component [component path]`
+- Component gets added to the list of declarations
+- Generated component follows the same structure as `AppComponent`
+- Prefix `app` on both components. This is an Angular best practice
+
+---
+
+## Integrating a Component
+
+Every component has a selector:
+
+*example.component.ts*
+```typescript
+@Component({
+	selector: 'app-example',
+	template: '<p>Multiple component example</p>'
+})
+export class ExampleComponent { }
+```
+
+---
+
+## Integrating a Component
+
+Use selector as an HTML tag within another component's template
+
+
+*app.component.ts*
+```typescript
+@Component({
+	selector: 'app-root',
+	template: `
+	  <h1>My Angular App</h1>
+	  <app-example></app-example>
+	`
+})
+export class AppComponent { }
+```
+[View Example](https://plnkr.co/edit/CMGx2TLUdPcL0D4tvGI9?p=preview)
+
+
+---
+
+## Conditional Rendering
+
+- Done using an Angular __directive__
+- __Directives__ are some logic that affects how some chunk of the page is rendered
+- To render conditionally, Angular provides the built in `ngIf` directive
+
+---
+
+## Directive Syntax
+
+- Used within a component's template
+- Invoked by using a data binding
+
+```html
+[target]="expression"
+```
+
+- The __target__ of a data binding is the directive
+- The __expression__ in a data binding is a JavaScript expression that produces a value (with some [exceptions](https://angular.io/docs/ts/latest/guide/template-syntax.html#!#template-expressions))
+
+---
+
+## Using ngIf
+
+```html
+<h1>Using ngIf</h1>
+<app-greet *ngIf="true"></app-greet>
+```
+[View Example](https://plnkr.co/edit/gVYY0CZU2qFvFlFXIB4m?p=preview)
+
+- Slightly different syntax. Using an asterisk
+- Can be bound to both components and plain HTML
+
+---
+
+## Handling Events
+
+We can bind an expression to any DOM event using the `(event)` syntax:
+
+```typescript
+@Component({
+  selector: 'app-counter',
+  template: `
+    <div>
+      <p>Count: {{num}}</p>
+      <button (click)="increment()">Increment</button>
+    </div>
+    `
+})
+export class CounterComponent {
+  num = 0;
+
+  increment() {
+    this.num++;
+  }
+}
+```
```

Comment:
Need more explanation of what this is doing - especially the automatic update of `{{num}}` in the HTML